controllers/admin/AdminPreferencesController.php

```diff
@@ -245,8 +245,9 @@ public function updateOptionPsMultishopFeatureActive($value)
     {
         Configuration::updateValue('PS_MULTISHOP_FEATURE_ACTIVE', $value);
 
-        $tab = Tab::getInstanceFromClassName('AdminShopGroup');
-        $tab->active = (bool)Configuration::get('PS_MULTISHOP_FEATURE_ACTIVE');
-        $tab->update();
+        if ($tab = Tab::getInstanceFromClassName('AdminShopGroup')) {
```

Comment:
Don't you want to check the tab exists with `(int)Tab::getIdFromClassName('AdminShopGroup');`?

`Tab::getInstanceFromClassName()` seems to return an instance of Tab whatever you send as params. If the tab doesn't exist, it will return an empty Tab anyway, and I guess you will pass the `if()` check and get the same result.

Reply:
@Quetzacoalt91 :+1:  yes! PR updated